scripts/gen_crops_from_obj_tracker_logs.py

```diff
@@ -54,7 +87,7 @@ def get_crops(mot_detections_file, imgs_path, out_dir, area_tol=1500):
         info = line.split(",")
         timestamp, obj_id, x, y, w, h = info[0], info[1], int(info[2]), int(
             info[3]), int(info[4]), int(info[5])
-        img = cv2.imread(imgs_path + "/carla-center-{}.png".format(timestamp))
+        img = cv2.imread(imgs_path + "/perfect-detector-{}.png".format(timestamp))
```

Comment:
Is it ok to use the perfect-detector images? Don't they contain the bounding boxes?